_blog/2018/aws-nlb.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Configure Istio Ingress with AWS NLB"
+overview: Describes how to configure istio ingress with an external load balancer NLB on AWS
+publish_date: April 08, 2018
+subtitle: Ingress AWS NLB
+attribution: Julien SENON
+
+order: 94
+
+
+layout: blog
+type: markdown
+redirect_from: "/blog/aws-nlb.html"
+---
+{% include home.html %}
+
+Instructions to use and configure ingress istio with `aws nlb`.
```

Comment:
perhaps you could add a link to AWS nlb here?  such as [aws nlb](link).

Please capitalize all occurrences of Istio used as a noun.